core/src/odbc_uri.rs

```diff
@@ -37,6 +37,7 @@ lazy_static! {
     .case_insensitive(true)
     .build()
     .unwrap();
+    static ref AUTH_SRC_REGEX: Regex = Regex::new(r"authSource=([a-zA-Z_0-9]*)").unwrap();
```

Comment:
[super nit/question] Why not using lazy_static? If a user never pass the URI we don't need to generate the regex, isn't it?

Reply:
If it's not in a `lazy_static` it's actually built at compile time, which is nice if we can do that. I thought this was in a lazy static, but I reverted all the authSource code, so I'm not sure.